app/controllers/application_records_controller.rb

```diff
@@ -10,10 +10,11 @@ def create
     create_user if @current_user.blank?
     data = parse_application_data(params.require :data)
     params.require :position_id
-    ApplicationRecord.create(params.permit(:position_id, :ethnicity, :gender)
-                             .merge(data: data,
-                                    user: @current_user,
-                                    reviewed: false))
+    record = ApplicationRecord.create(record_params
```

Comment:
The indentation is a little awkward here. I'd suggest constructing the params in an intermediate variable before the call to `ApplicationRecord.create`.